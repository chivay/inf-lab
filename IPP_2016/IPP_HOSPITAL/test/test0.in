NEW_DISEASE_ENTER_DESCRIPTION Kowalski Bardzo ciężka choroba
NEW_DISEASE_ENTER_DESCRIPTION Nowak delikatne przeziębienie...
NEW_DISEASE_COPY_DESCRIPTION Kowalski nowak
NEW_DISEASE_COPY_DESCRIPTION Kowalski Nowak
PRINT_DESCRIPTION Kowalski 3
PRINT_DESCRIPTION Kowalski 2
CHANGE_DESCRIPTION Kowalski 2 przeziębienie z powikłaniami !
NEW_DISEASE_COPY_DESCRIPTION van_Beethoven Kowalski
NEW_DISEASE_COPY_DESCRIPTION van_Beethoven Nowak
DELETE_PATIENT_DATA Kowalski
NEW_DISEASE_COPY_DESCRIPTION Nowak Kowalski
PRINT_DESCRIPTION van_Beethoven 1