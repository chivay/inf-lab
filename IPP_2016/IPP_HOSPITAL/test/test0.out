OK
OK
IGNORED
OK
IGNORED
delikatne przeziębienie...
OK
OK
OK
OK
IGNORED
przeziębienie z powikłaniami !
